mkat_fpga_tests/test_cbf.py

```diff
@@ -480,9 +480,9 @@ def get_actual_phases():
                 # correlator_fixture.katcp_rct.req.delays time.time+somethign
                 # See page 22 on ICD ?delays on CBF-CAM ICD
                 reply = correlator_fixture.katcp_rct.req.input_labels()
```

Comment:
You should probably change this to be reply, informs = correl......, then you can also skip the [0] you added below.